frontend/.husky/commit-msg

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+cd frontend && yarn commitlint --edit $1
```

Comment:
Let's use npm here. Why?
Because it will break if someone want to contribute and doesn't have yarn installed.

Reply:
there are other in pre-commit stage let's cover in another PR